guides/micronaut-data-r2dbc-repository/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "Access a database with Micronaut Data R2DBC",
+  "intro": "Learn how to access a database with Micronaut R2DBC repositories.",
+  "authors": ["Sergio del Amo", "John Shingler"],
```

Comment:
add your name

Reply:
I added @graemerocher as the author  https://github.com/micronaut-projects/micronaut-guides/pull/965/commits/201a7b0b15ffe785f9d3115acdb4a5dd60361be6